.gitignore

```diff
@@ -0,0 +1,4 @@
+/node_modules
```

Comment:
You need to change that for `/server/node_modules` and delete the node_modules folder.